### PR TITLE
[OYPD-250] Facility branch.

### PR DESCRIPTION
**Pull request by Kerby70**
Jira https://propeople-us.atlassian.net/browse/OYPD-250
D.o https://www.drupal.org/node/2852681

Steps to review
- [x] Create a Branch with required information.
- [x] Create a Facility with required information, referencing the new Branch in the field "Facility Branch"
- [x] View the Facility node page
- [x] See in the sidebar under the heading "ASSOCIATED BRANCH" a render of the teaser display of the Branch referenced.

### CODE REVIEW

**Comment by gundevel** [comment]
Build comment file: 
--------------
===============
Build site installed at http://openy-dev.ffwua.com/build468

--------------
Test PASSed.
Refer to this link for build results (access rights to CI server needed): 
http://openy-dev.ffwua.com:8080/job/PR_BUILDER/468/

**Comment by gundevel** [comment]
Behat test results file URL: http://openy-dev.ffwua.com/build468/profiles/openy/build/reports/behat

**Review by ddrozdik — APPROVED** [review]

**Comment by ddrozdik** [comment]
retest this please

**Comment by bjwilkins** [comment]
Build comment file: 
--------------
===============
Build site installed at http://openy-dev.ffwua.com/build470

--------------
Test PASSed.
Refer to this link for build results (access rights to CI server needed): 
http://openy-dev.ffwua.com:8080/job/PR_BUILDER/470/

**Comment by bjwilkins** [comment]
Behat test results file URL: http://openy-dev.ffwua.com/build470/profiles/openy/build/reports/behat

**Comment by ddrozdik** [comment]
Added branch and facility page, but block with the branch info  looks differ than on the screenshot in the task. 
http://openy-dev.ffwua.com/build470/facility/facilty

File: modules/openy_features/openy_location/modules/openy_loc_facility/config/install/core.entity_view_display.node.facility.default.yml [hunk 1]
```diff
@@ -33,6 +34,14 @@ content:
       link: ''
     third_party_settings: {  }
     type: entity_reference_revisions_entity_view
+  field_facility_loc:
+    weight: 113
+    label: above
+    settings:
+      view_mode: teaser
```
Comment (ddrozdik) [inline modules/openy_features/openy_location/modules/openy_loc_facility/config/install/core.entity_view_display.node.facility.default.yml:41]: Seem here should be another view mode

Comment (alexschedrov) [reply modules/openy_features/openy_location/modules/openy_loc_facility/config/install/core.entity_view_display.node.facility.default.yml:41]: @ddrozdik it's fine. Theming will be applied in separate issue.

**Review by alexschedrov — APPROVED** [review]
Approved.

**Pull request merged**